props/saber_fett263_buttons.h

```diff
@@ -1694,23 +1698,25 @@ SaberFett263Buttons() : PropBase() {}
   };
 
   void StartTrackPlayer() {
-    track_num_ = 0;
     num_tracks_ = RunCommandAndGetSingleLine("list_current_tracks", nullptr, 0, 0, 0);
     if (num_tracks_ > 0) {
       sound_library_.SaySelect();
       StartMenu(MENU_TRACK_PLAYER);
+      track_num_ = 1;
+      PlayTrack();
     } else {
+      // Loop default track if tracks not found
       sound_library_.SayLoop();
+      track_num_ = 0;
       track_mode_ = PLAYBACK_LOOP;
+      StartOrStopTrack();
     }
-    StartOrStopTrack(); 
   }
 
   void TrackPlayer() {
     if (track_mode_ != PLAYBACK_OFF) {
       if (!track_player_) {
-#ifdef ENABLE_AUDIO
-        if (track_num_ >= 0 && track_mode_ == PLAYBACK_LOOP) {
+        if (track_num_ <= 0 && track_mode_ == PLAYBACK_LOOP) {
```

Comment:
It seems weird to test both track_num_ and track_mode_ here.
It seems to me that just checking track_num_ == 0 would be enough?

Reply:
I have to check for PLAYBACK_LOOP otherwise the default track just keeps playing itself no matter what the user does if they don't have other tracks in font because TrackPlayer() is in Loop() and will run StartorStopTrack()

Reply:
ok, but it looks weird